source/NVDAObjects/UIA/edge.py

```diff
@@ -508,13 +508,44 @@ def event_gainFocus(self):
 			return
 		return super(EdgeHTMLRootContainer,self).event_gainFocus()
 
+class EdgeHeadingQuickNavItem(UIATextRangeQuickNavItem):
+
+	@property
+	def level(self):
+		if not hasattr(self,'_level'):
+			styleVal=getUIATextAttributeValueFromRange(self.textInfo._rangeObj,UIAHandler.UIA_StyleIdAttributeId)
+			self._level=styleVal-(UIAHandler.StyleId_Heading1-1) if UIAHandler.StyleId_Heading1<=styleVal<=UIAHandler.StyleId_Heading6 else None
+		return self._level
+
+	def isChild(self,parent):
+		return self.level>parent.level
+
+def EdgeHeadingQuicknavIterator(itemType,document,position,direction="next"):

```

Comment:
Sorry I meant on `EdgeHeadingQuicknavIterator`. As an aside, how would you feel about adding doc strings on derived class functions to say something like `@see parentclass.methodname`. This would make it clearer that this is an override, and where to find the documentation for it.